assets/js/components/Dashboard/AlertsPage.tsx

```diff
@@ -205,7 +205,7 @@ const AlertsList: ComponentType<AlertsListProps> = ({
 
       return screensWithAlert
         ? screensWithAlert.find((screen_id) =>
-            ids.includes(screenMetaData[screen_id].type),
+            ids.includes(screenMetaData?.[screen_id]?.type),
```

Comment:
When I looked a little deeper into this it seemed a little worrisome, I fixed it here so it doesn't straight up crash but it seems like some of the `screen_ids` might be missing. 

I don't have the full context of the underlying cause of this, but I can take a deeper peek if we feel it's necessary and it's not a known issue.

Reply:
Was this related to the name change in some way? Did you run into the crash you mentioned while testing that change locally? Curious why TypeScript wouldn't have caught this.

Reply:
Yeah let me double check - I just checked `main` and it seemed like it was a problem, but dev/prod look fine. It might be my own outdated/incorrect local data. 

That being said, I'll revert this.